package.json

```diff
@@ -4,22 +4,35 @@
   "description": "Note blogging",
   "main": " ",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "clean": "rm -rf ./static/build/js",
+    "watch": "nodemon --watch ./static/js/ -e js,jsx --exec 'browserify -t babelify -e ./static/js/app/App.jsx > ./static/build/js/app.js --debug'",
+    "lint": "eslint ./static/js/app/*.jsx",
+    "prebuild": "npm run clean && mkdir -p ./static/build/js/",
+    "build": "browserify -t babelify -e ./static/js/app/App.jsx | uglifyjs -c > ./static/build/js/app.min.js"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/alvarohurtado84/notenote.git"
   },
-  "author": "Alvaro Hurtado Mochon @alvarolab",
+  "author": "Alvaro Hurtado Mochon <alvarohurtado84@gmail.com> (http://www.alvarohurtado.es/)",
```

Comment:
Just realized while I'm very sleepy that you can add your twitter back if you like instead of your website `https://twitter.com/alvarolab`